test/entrypoint/incorrect_roles_sequence.yaml

```
@@ -0,0 +1,27 @@
+credentials:
+  users:
+    guest:
+      roles: [super]
+
+groups:
+  group-001:
+    replicasets:
+      replicaset-001:
+        instances:
+          master:
+            roles: [roles.metrics-export, roles.httpd]
```

```suggestion
            roles: [roles.metrics-export]
```

What will happen in this case?

---

The same error that was introduced in this PR will occur.